src/training/validator.h

```diff
@@ -193,15 +193,11 @@ class CrossEntropyValidator : public Validator<data::Corpus> {
 
 class ScriptValidator : public Validator<data::Corpus> {
 public:
-  ScriptValidator(std::vector<Ptr<Vocab>> vocabs, Ptr<Config> options)
+  ScriptValidator(std::vector<Ptr<Vocab>> vocabs, Ptr<Options> options)
       : Validator(vocabs, options, false) {
-    Ptr<Options> opts = New<Options>();
-    opts->merge(options);
-    opts->set("inference", true);
```

Comment:
Just to be sure, inherited from above? Maybe there should be `ABORT_IF(!options->get<bool>("inference", false), ...)` in every validator.

Reply:
Agree. Adding this to Validator should be enough, though.